packages/inference/test/vcr.ts

```diff
@@ -0,0 +1,163 @@
+import { isBackend, isFrontend } from "../src/utils/env-predicates";
+
+const TAPES_FILE = "./tapes.json";
+
+let VCR_MODE;
```

Comment:
It should be typed (I guess `tsconfig.json` or `eslint` should be configured to pick this up)